docs/markdown/android/icon.md

```diff
@@ -6,6 +6,10 @@ fullwidth: true
 <ImgContainer src="https://i.pinimg.com/originals/5c/e9/65/5ce965aac16951ef57b3a735bb41e823.jpg" alt="Four example icons in a row. Icons include share, check, back, and menu." />
 
 
+## Icon assets
+
+Download svg assets from our [Android icon GitHub repo](packages/gestalt/src/icons-android).
```

Comment:
svg >> SVG typo

This link goes nowhere.

Reply:
Android icon GitHub repo what is this?

Reply:
a temporary folder for icons until they can automate the process